addons/website_popup/static/src/css/website_popup.css

```diff
@@ -0,0 +1,6 @@
+@charset "UTF-8";
+/* Summernote not Support for placeholder text https://github.com/summernote/summernote/issues/581 */
+.editor_enable [data-oe-placeholder]:empty:not(:focus):before {
+  content: attr(data-oe-placeholder);
+  opacity: 0.3
+}
```

Comment:
this is a generic issue ,so it is better to move inside editor.css (website module).